acra-translator/go-translator-demo/cmd/api/.env.example

```diff
@@ -0,0 +1,10 @@
+ADDRESS=0.0.0.0:8008
```

Comment:
can you explain where we use this file or how it should be used? I don't see usage in the dockerfile or mounting in docker-compose files, and don't see mention in the readme